fuzz/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+target
+libfuzzer
```

Comment:
With the shared workspace (see other comment) there shouldn’t be a `target` directory here. As to `libfuzzer`, I’d prefer it to be `fuzz/libfuzzer` in `.gitignore` at the root of the repo.